android/src/main/java/com/swmansion/rnscreens/SearchViewFormatter.kt

```diff
@@ -10,6 +10,7 @@ import androidx.appcompat.widget.SearchView
 class SearchViewFormatter(var searchView: SearchView) {
     private var mDefaultTextColor: Int? = null
     private var mDefaultTintBackground: Drawable? = null
+    private var mDefaultText: String = ""
```

Comment:
Is default text needed there? I believe `placeholder` is being used as a default text - if this prop is not being served as a Search Bar prop, let's remove it.